### DISCUSSION TITLE
Is this behaviour in DateRangeInput2 intended or a bug? 🐛

### DISCUSSION THREAD

**Discussion by yasmin-e** [palantir/blueprint#6052] [Q&A]
When you "go backwards", it seems to extend the date range in that direction.

In this gif I've set a range in the future, and it's really hard to change my range to just be a single date in the past. This might be the intended behavior, but I found it unintuitive.

I also found out while playing around with the component in the docs, this only happens when the format includes a time - if it's just a date, the default is to extend the range forward.

![228893207-9442d82f-e13f-4a3b-8351-beaf878bccca](https://user-images.githubusercontent.com/67699259/229534228-08aadeab-0eb1-4567-b928-e48f3926e3a4.gif)

**Comment by adidahiya** [comment]
@yasmin-e It looks like your use case works fine in DateRangePicker (which DateRangeInput2 uses under the hood):

![2023-04-19 14 47 28](https://user-images.githubusercontent.com/723999/233171466-ad6f112b-dd77-4035-ad3d-480565e393c4.gif)

DateRangeInput2 has additional input focus behavior which changes the UX here, but it should still be possible to select a single day in the past:

![2023-04-19 14 51 18](https://user-images.githubusercontent.com/723999/233172191-8c579aab-7e33-4a59-8f71-3dc4daba6744.gif)

I think it depends on which end of the range you start selecting first, or which of the inputs you have focused when opening the popover. I'm definitely open to any PRs which make the UX more intuitive here.
